### ISSUE TITLE
NTR: 3-hydroxyisobutyryl-CoA hydrolase deficiency (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8192]
New term request 3-hydroxyisobutyryl-CoA hydrolase deficiency (segal_070716194832): HIBCH; beta-hydroxyisobutyrl-CoA deacylase

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000215
- label: Reduced circulating 3-hydroxyisobutyryl-CoA hydrolase activity
- parents: [Abnormal circulating enzyme concentration or activity (HP:0012379)](https://hpo.jax.org/app/browse/term/HP:0012379)
- pmid: PMID:17160907
- def: The activity of 3-hydroxyisobutyryl-CoA hydrolase in the blood corculation is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8192
- synonyms: 3-hydroxyisobutyryl-CoA hydrolase deficiency(NONE)
